oak_functions_launcher/build.rs

```diff
@@ -36,10 +37,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             "{}oak_functions_service/proto/oak_functions.proto",
             env!("WORKSPACE_ROOT")
         )],
-        &[format!(
-            "{}oak_functions_service/proto",
-            env!("WORKSPACE_ROOT")
-        )],
+        &[env!("WORKSPACE_ROOT")],
```

Comment:
Did you confirm that this is not rebuilt for every unrelated change in the workspace?

Reply:
Not sure, I think it still recompiles

I.e. `oak_client` crate is being built in `0.12s`, but when I change an unrelated `oak_functions.proto` it becomes `0.21s`

Reply:
Please make sure it does not recompile everything, and check properly (not just based on time). You can look at the output of cargo build and check whether it rebuilds this package or not.

Reply:
Looks like it doesn't recompile:
```bash
$ cargo build --package=oak_client --release
    Finished release [optimized] target(s) in 0.13s
$ cargo build --package=oak_client --release
    Finished release [optimized] target(s) in 0.15s
```
```bash
$ cargo build --package=oak_functions_service
    Finished dev [unoptimized + debuginfo] target(s) in 0.13s
$ cargo build --package=oak_functions_service
    Finished dev [unoptimized + debuginfo] target(s) in 0.13s
```